src/app/wallet/views/gas.nim

```diff
@@ -52,22 +52,21 @@ QtObject:
     result.setup
 
   proc getGasEthValue*(self: GasView, gweiValue: string, gasLimit: string): string {.slot.} =
-    var gweiValueInt:int
     var gasLimitInt:int
 
-    discard gweiValue.parseInt(gweiValueInt)
     discard gasLimit.parseInt(gasLimitInt)
 
     # The following two check prevents app crash, cause we're trying to promote 
     # gweiValueInt and gasLimitInt to unsigned 256 int, and these two numbers
     # must be positive numbers, because of overflow.
```

Comment:
```suggestion
    # The following check prevents app crash, cause we're trying to promote 
    # gasLimitInt to unsigned 256 int, and this number must be a positive number,
    # because of overflow.
```